pkg/controller.v1/tensorflow/controller.go

```diff
@@ -70,6 +71,8 @@ const (
 
 	TFJobEvictAnnotation = "cluster-autoscaler.alibabacloud.com/evict-for-failed-pod"
 	PodEvictAnnotation   = "cluster-autoscaler.kubernetes.io/safe-to-evict"
+
+	TFJobWaitingWorkerAnnotation = "arena.runningPod.ttlSecondsAfterFinished"
 )
```

Comment:
I suggest renaming to `arena.kubeflow.org/pod.ttlSecondsAfterFinished`.

Reply:
Done